### ISSUE TITLE
Tab can't move past an open dropdown

### ISSUE THREAD

**Issue by mbrakken** [bl00mber/react-phone-input-2#282]
When the dropdown is open and focused, it becomes somewhat of a keyboard trap. Users cannot tab past it, and the only keyboard accessible ways of closing it are enter and esc, both of which shift focus backwards to the text box. Additionally, enter selects the active value, so it doesn't serve as an exit method.

WCAG 2.0 No Keyboard Trap states:
> If keyboard focus can be moved to a component of the page using a keyboard interface, then focus can be moved away from that component using only a keyboard interface, and, if it requires more than unmodified arrow or tab keys or other standard exit methods, the user is advised of the method for moving focus away. (Level A)

https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-trapping.html

Given that esc moves back to the input, it doesn't feel it fulfills the requirement of moving focus away.
